code-name-taran/src/com/spaghettisoft/component/menu/HangmanMenu.java

```diff
@@ -0,0 +1,14 @@
+package com.spaghettisoft.component.menu;
+
+import java.util.List;
+
+import com.spaghettisoft.component.menu.items.MenuItem;
+
+public class HangmanMenu extends Menu {
+
+	public HangmanMenu(List<MenuItem> menuItems, String exitOptionName) {
+		super(menuItems, exitOptionName);
+		// TODO Auto-generated constructor stub

```

Comment:
Remember to remove auto-generated comments. You can also remove the auto-generated comments from the options in eclipse.